source/unified-test-format/tests/valid-pass/createEntities-operation.yml

```diff
@@ -0,0 +1,55 @@
+description: createEntities-operation
+
+# Note: createEntities is not technically in the 1.9 schema but was introduced at the same time.
+schemaVersion: "1.9"
+
+createEntities:
+  - client:
+      id: &client0 client0
+  - database:
+      id: &database0 database0
+      client: *client0
+      databaseName: &database0Name database0
+  - collection:
+      id: &collection0 collection0
+      database: *database0
+      collectionName: &collection0Name coll0
+
+initialData:
```

Comment:
Conceivably, you could delete the top-level `createEntities` and `initialData` blocks in this test. They don't appear to be used for anything.

Reply:
Done.